2024Robot_Team3484/src/main/cpp/subsystems/IntakeSubsystem.cpp

```diff
@@ -0,0 +1,92 @@
+//
+// Intake Subsystem
+//
+// -Noah, Aidan  & Ethan
+//
+
+#include <subsystems/IntakeSubsystem.h>
+#include <units/angle.h>
+
+using namespace IntakeConstants;
+
+IntakeSubsystem::IntakeSubsystem() {
+    _pivot_motor.RestoreFactoryDefaults();
+    _drive_motor.RestoreFactoryDefaults();
+
+    _drive_motor.SetIdleMode(rev::CANSparkMax::IdleMode::kBrake);
+
+    _pivot_pid_controller.SetFeedbackDevice(_pivot_encoder);
+
+    _target_position = STOW_POSITION;
+
+    _pivot_pid_controller.SetP(0.1);
+    _pivot_pid_controller.SetI(1e-4);
+    _pivot_pid_controller.SetD(1);
+    _pivot_pid_controller.SetIZone(0);
+    _pivot_pid_controller.SetFF(0);
+    _pivot_pid_controller.SetOutputRange(-1, 1);
```

Comment:
I suggest you convert all of these hardcoded values in to constants (`#define`) in `Constants.h`

Reply:
I opted to use `constexpr` as I'm not sure if using `#define` will be able to passed as a constructor. Plus, the other constants are using `constexpr` and that makes more sense to me.